src/policy/policy.h

```diff
@@ -158,11 +158,6 @@ int64_t GetVirtualTransactionSize(int64_t nWeight, int64_t nSigOpCost, unsigned
 int64_t GetVirtualTransactionSize(const CTransaction& tx, int64_t nSigOpCost, unsigned int bytes_per_sigop);
 int64_t GetVirtualTransactionInputSize(const CTxIn& tx, int64_t nSigOpCost, unsigned int bytes_per_sigop);
 
-static inline int64_t GetVirtualTransactionSize(const CTransaction& tx)
```

Comment:
Alternative, move to wallet and rename to `GetWalletVirtualTransactionSize` with tests/assertions/comments saying why it's safe in one location instead of spread around? Would make grepping for the remaining bugs easier as well. e.g., anything like `GetVirtualTransactionSize(tx, 0, 0)` would be obviously a bug at that point